cdap-docs/admin-manual/source/upgrading/cloudera.rst

```diff
@@ -44,6 +44,12 @@ CDAP:
 
 #. Stop all flows, services, and other programs in all your applications.
 
+#. If you are upgrading an *authorization-enabled* CDAP instance, you will need to give 
+   the *cdap* user *ADMIN* privileges on all existing CDAP namespaces. **Note:** the *ADMIN*
+   privilege does not give the *cdap* user *READ* or *WRITE* privileges on the namespaces.
+   As these are temporary grants required only for the purpose of running the upgrade tool, 
+   once the upgrade is complete, these grants can be revoked.
+
```

Comment:
"Alternatively, you can also disable authorization before running the upgrade tool, and enable it again after the upgrade is completed."?

Also, perhaps a link to the authorization admin manual?